src/rgba.rs

```diff
@@ -2,90 +2,76 @@
 // See the COPYRIGHT file at the top-level directory of this distribution.
 // Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
 
-use glib::translate::*;
 use ffi;
-use ffi::GdkRGBA;
+use glib::translate::*;
+use glib_ffi::gconstpointer;
+use std::fmt;
+use std::mem;
+use auto::RGBA;
 
-pub trait RGBA {
-    fn white() -> GdkRGBA;
-    fn blue() -> GdkRGBA;
-    fn green() -> GdkRGBA;
-    fn red() -> GdkRGBA;
-    fn black() -> GdkRGBA;
-    fn copy(&self) -> GdkRGBA;
-    fn parse(&mut self, spec: &str) -> bool;
-    fn to_string(&self) -> Option<String>;
-}
+impl RGBA {
 
-impl RGBA for GdkRGBA {
-    fn white() -> GdkRGBA {
+    fn new(red: f64, green: f64, blue: f64, alpha: f64) -> RGBA {
         skip_assert_initialized!();
-        GdkRGBA {
-            red: 1f64,
-            green: 1f64,
-            blue: 1f64,
-            alpha: 1f64
+        ffi::GdkRGBA {
+            red: red,
+            green: green,
+            blue: blue,
+            alpha: alpha,
         }
```

Comment:
Yeah, `new()` currently doesn't compile and I'm uncertain how to actually implement it. Should it be something like:
```
let mut rgba = RGBA::uninitialized();
rgbba.deref_mut().red = red;
...
```

That doesn't compile for me, but I'm not really sure how else to create a new `GdkRGBA` struct and embed it in the `auto::RGBA` type.